CI/setup-windows-sdk.sh

```diff
@@ -284,4 +284,12 @@ cd ~ || exit 1
 echo "  ... setup-windows-sdk.sh shell script finished."
 echo ""
 
+echo "Copy the following lines into the build environment for a project in Qt Creator:"
+echo "See https://doc.qt.io/qtcreator/creator-how-set-project-environment.html#change-the-environment-for-a-project"
+echo ""
+MSYS_ROOT=$(cygpath -aw /)
+echo "MINGW_BASE_DIR=${MSYS_ROOT}$(echo ${MSYSTEM_PREFIX} | sed 's/\//\\/g')"
+echo "LUA_PATH=$(luarocks --lua-version 5.1 path --lr-path)"
+echo "LUA_CPATH=$(luarocks --lua-version 5.1 path --lr-cpath)"
```

Comment:
Unfortunately you cannot use `cygpath`s `-p` option to clean up a path list (with the "mixed" mode forward directory separators but with the Windows colon suffixed drive specifier as it is predicated around a `:` separator and not the `;` that Lua uses and `:` become ambiguous) - instead we have to use `sed` to make all the "toothpicks" lean forward like a `cygpath -m` would do...

You also need to include the `-tree "user"` option for end-users/hackers if they are using the recommend (by me in my original scripts) way to have a "per-user" rocks installation {`--local` doesn't work on MSYS2+Mingw64}...
```suggestion
echo "LUA_PATH=$(luarocks --lua-version 5.1 --tree "user" path --lr-path)" | sed "s|\\\\|/|g"
echo "LUA_CPATH=$(luarocks --lua-version 5.1 --tree "user" path --lr-cpath)" | sed "s|\\\\|/|g"
```

See the differences:
```bash
stephen@Rachel MINGW64 ~/src/mudlet
$ echo "LUA_PATH=$(luarocks --lua-version 5.1 path --lr-path)"
LUA_PATH=C:\msys64\home\stephen/.luarocks-MINGW64/share/lua/5.1/?.lua;C:\msys64\home\stephen/.luarocks-MINGW64/share/lua/5.1/?/init.lua;C:/msys64/mingw64/share/lua/5.1/?.lua;C:/msys64/mingw64/share/lua/5.1/?/init.lua

stephen@Rachel MINGW64 ~/src/mudlet
$ echo "LUA_PATH=$(luarocks --lua-version 5.1 path --lr-cpath)"
LUA_PATH=C:\msys64\home\stephen/.luarocks-MINGW64/lib/lua/5.1/?.dll;C:/msys64/mingw64/lib/lua/5.1/?.dll

stephen@Rachel MINGW64 ~/src/mudlet
$ echo "LUA_PATH=$(luarocks --lua-version 5.1 --tree "user" path --lr-path)" | sed "s|\\\\|/|g"
LUA_PATH=C:/msys64/home/stephen/.luarocks-MINGW64/share/lua/5.1/?.lua;C:/msys64/home/stephen/.luarocks-MINGW64/share/lua/5.1/?/init.lua;C:/msys64/home/stephen/.luarocks-MINGW64/share/lua/5.1/?.lua;C:/msys64/home/stephen/.luarocks-MINGW64/share/lua/5.1/?/init.lua;C:/msys64/mingw64/share/lua/5.1/?.lua;C:/msys64/mingw64/share/lua/5.1/?/init.lua

stephen@Rachel MINGW64 ~/src/mudlet
$ echo "LUA_PATH=$(luarocks --lua-version 5.1 --tree "user" path --lr-cpath)" | sed "s|\\\\|/|g"
LUA_PATH=C:/msys64/home/stephen/.luarocks-MINGW64/lib/lua/5.1/?.dll;C:/msys64/home/stephen/.luarocks-MINGW64/lib/lua/5.1/?.dll;C:/msys64/mingw64/lib/lua/5.1/?.dll

```

Reply:
OK... this is getting too complex, and I don't foresee that many developers making use of this feature. Let's move it out of the scripts and into a wiki page for those who are interested in this.

Reply:
Ah no, this is necessary and I was thinking of something else. 

I prefer to keep it this way because it worked for me and others in testing.

Reply:
Well what I use works for me:
![image](https://github.com/user-attachments/assets/419fe078-4a8f-4f38-9008-2a30bc7353ba)

The mixed directory separators are confusing should they get presented to the user in a red/yellow error message when the Lua subsystem fails to start up correctly. As such I took me a while to find out what would work and `/` as a separator needs much less - if any - escaping...

Sometimes Windows might not let you install rocks into a "sub-system-wide" installation but a per-user one is more likely to work - for anyone with a PC with more than one user - a strange concept these days I know - but it does happen!

Reply:
Oh, forget the `--tree "user"` it seems to not be needed - which is weird! Look how it seemed to double up the existing stuff. :confounded: Anyhow - the conversion of all the directory separators to be `/` still looks to be helpful - IMHO.

Reply:
Okay - I am confused and not sure what are you asking for, sorry! I tested the PR locally before submitting it and everything was working fine for me.

Have you tested the PR and found that it does not work for you?